ramls/check-in.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Check-in operation",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": "string",
+      "description": "Unique ID (generated UUID) of the check-in operation",
+      "$ref": "raml-util/schemas/uuid.schema"
+    },
+    "occurredDateTime": {
+      "type": "string",
+      "description": "Date and time when the check-in has been occurred",
+      "format": "date-time"
+    },
+    "itemId": {
+      "type": "string",
+      "description": "Id of item that has checked in",
+      "$ref": "raml-util/schemas/uuid.schema"
+    },
+    "checkInServicePointId": {
```

Comment:
I think this can be `servicePointId` as it is in the context of a check in record. What do you think?

Reply:
Renamed to `servicePointId`